modules/ROOT/pages/_partials/common-file-attributes.adoc

```diff
@@ -1,4 +1,4 @@
 == Attributes
 // Included in sftp-connector.adoc, ftp-connector.adoc, file-connector.adoc
 
-When reading or listing files, you might be interested in the file's metadata (for example, the file name, full path, size, timestamp, and so on). The connector uses the Mule Message Attributes to access this information. More information about file attributes are available in the <<see_also, reference>> for this connector.
\ No newline at end of file
+When reading or listing files, the connector provides file metadata such as, the file name, full path, size, timestamp, and so on. The connector uses the Mule Message Attributes to access this information. 
```

Comment:
"metadata such as, the" -> "metadata, such as the"
see https://www.chicagomanualofstyle.org/qanda/data/faq/topics/Commas/faq0025.html

Reply:
Fixed